nodejs/week3/week3/warmup/index.js

```diff
@@ -0,0 +1,99 @@
+const express = require("express");
+const app = express();
+app.use(express.urlencoded({extended:false}));
+app.use(express.json());
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+
+app.get("/calculator/multiply", (req, res) => {
+   
+    const firstNumber= Number(req.query.firstParam);
+    const secondParam=req.query.secondParam;
+   
+    
+    if(typeof(secondParam) === 'object'){
+       
+        const secondParamArray=secondParam.reduce( function (previous,current) {
+         return Number(previous)*Number(current)});
+        
+         const result1=firstNumber * secondParamArray;
+        res.json(`The answer for your request is ${result1}`);
+    }else{
+        const secondNumber=Number(req.query.secondParam);
+        const result =firstNumber * secondNumber;
+        res.json(`The answer for your request is ${result}`);
+    }   
+   
+});
+
+app.get("/calculator/add", (req, res) => {
+   
+    const firstNumber= Number(req.query.firstParam);
+    const secondNumber=Number(req.query.secondParam);
+    const thirdNumber =Number(req.query.thirdParam);
```

Comment:
what if there is a fourth number?

I think you should use the array method reduce to add all secondNumber like you did above.